components/posts/Comment.tsx

```diff
@@ -1,59 +1,68 @@
 import React from "react";
 import { View, Text, StyleSheet, Dimensions } from "react-native";
-import { ThemedIconButton } from "@/components/theme/ThemedIconButton"; 
-
+import { ThemedIconButton } from "@/components/theme/ThemedIconButton";
 
 const { width, height } = Dimensions.get("window");
 
 export type CommentType = {
-    comment: string;
-    user: string;
-    createdAt?:  string;
+  comment: string;
+  user: string;
+  createdAt?: string;
 };
 
-export function SingleComment(comment : CommentType) {
-
-    return (    
-        
-        <View style = {{flexDirection: 'row', justifyContent: 'space-between', paddingBottom:15, alignItems:'center'}}>
-            <ThemedIconButton
-            iconName="person-circle-outline"
-            onPress={() => {
-              /* user button */
-            }}
-            size={45}
-            color="white"
-          />
-        <View style = {styles.container}>
-            <View style = {{flexDirection: 'row', justifyContent: 'space-between', paddingBottom:10}}>
-                <Text style = {styles.user}> {comment.user} </Text>
-                <Text style = {styles.textofcomment}> {comment.createdAt} </Text>
-            </View>
-        <Text style = {styles.textofcomment}> {comment.comment} </Text>
-        </View>
+export function SingleComment(comment: CommentType) {
+  return (
+    <View
```

Comment:
Why not use a `ThemedView` here ?

Reply:
Actually as this code was produced while I was working on the branch, it wasn't in the scope of the refactoring, I think it will be produced in a similar ulterior branch. But you're right and I'll bring this up during team meeting.